conf/machine/raspberrypi4.conf

```diff
@@ -11,7 +11,8 @@ MACHINE_EXTRA_RRECOMMENDS += "\
     bluez-firmware-rpidistro-bcm4345c0-hcd \
 "
 
-SDIMG_KERNELIMAGE ?= "kernel7.img"
+# 'l' stands for LPAE
+SDIMG_KERNELIMAGE ?= "kernel7l.img"
```

Comment:
can we use LPAE for older rpi as well ?

Reply:
Most probably for RPi3 yes. But I'll let that testing be part of another PR.